.eslintrc

```diff
@@ -47,6 +50,21 @@
 		// TS rules
 		"@typescript-eslint/prefer-optional-chain": "error",
 		"@typescript-eslint/no-unused-vars": "error",
-		"@typescript-eslint/no-unused-expressions": "error"
+		"@typescript-eslint/no-unused-expressions": "error",
+		"@typescript-eslint/no-unnecessary-type-arguments": "error",
+		"@typescript-eslint/no-unnecessary-type-assertion": "error",
+		"@typescript-eslint/no-unnecessary-type-constraint": "error",
+		"@typescript-eslint/no-useless-constructor": "error",
+		"@typescript-eslint/no-useless-empty-export": "error",
+		"@typescript-eslint/prefer-enum-initializers": "error",
+		"@typescript-eslint/no-unnecessary-qualifier": "error",
+		"@typescript-eslint/prefer-for-of": "error",
+		"@typescript-eslint/prefer-includes": "error",
+		"@typescript-eslint/prefer-nullish-coalescing": "error",
+		"@typescript-eslint/prefer-string-starts-ends-with": "error",
+		"@typescript-eslint/sort-type-union-intersection-members": "error",
```

Comment:
I don't really like this rule, because I often want to use some more meaningful sorting instead of alphabetical.

Reply:
Do you want me to just get rid of the rule or also to revert the sorting changes?

Reply:
Well, you can keep the changes and delete just the rule. I think this will be easier than reverting the changes.)